test/extended/builds/clone_git_protocol.go

```diff
@@ -0,0 +1,51 @@
+package builds
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-builds][Feature:Builds] clone repository using git:// protocol", func() {
+	var (
+		oc = exutil.NewCLI("build-pruning")
+	)
+
+	g.Context("", func() {
+
+		g.BeforeEach(func() {
+			exutil.PreTestDump()
+		})
+
+		g.JustBeforeEach(func() {
+			g.By("waiting for openshift namespace imagestreams")
+			err := exutil.WaitForOpenShiftNamespaceImageStreams(oc)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+		})
+
+		g.AfterEach(func() {
+			if g.CurrentGinkgoTestDescription().Failed {
+				exutil.DumpPodStates(oc)
+				exutil.DumpConfigMapStates(oc)
+				exutil.DumpPodLogsStartingWith("", oc)
+			}
+		})
+
+		g.It("should clone using git:// if no proxy is configured", func() {
+			proxyConfigured, err := exutil.HasProxy(oc)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			if proxyConfigured {
+				fmt.Fprintf(g.GinkgoWriter, "Skipping test because proxy is configured")
+				return
+			}
+
+			g.By("creating a new application using the git:// protocol")
+			err = oc.Run("new-app").Args("git://github.com/openshift/ruby-hello-world.git").Execute()
```

Comment:
Historically we've had issues with `openshift/ruby-hello-world` - the scl example is a bit more stable:

```suggestion
			err = oc.Run("new-app").Args("git://github.com/sclorg/ruby-ex.git").Execute()
```

Reply:
updated in next push